src/common/progressBar.js

```diff
@@ -0,0 +1,73 @@
+/* @flow strict-local */
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+import Bar from 'react-native-progress/Bar';
+import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
+import { getLoading } from '../selectors';
+import { BRAND_COLOR, NAVBAR_SIZE } from '../styles';
+import { connect } from '../react-redux';
+import type { Dispatch } from '../types';
+
+type SelectorProps = $ReadOnly<{|
+  loading: boolean,
+|}>;
+
+type Props = $ReadOnly<{|
+  animated: boolean,
+  progress: number,
+  borderRadius: number,
+  borderWidth: number,
+  viewStyle: ViewStyleProp,
+  color: string | number,
+  unfilledColor: string,
+  height: number,
+  width: number | null,
+  indeterminate: boolean,
+  indeterminateAnimationDuration: number,
+  useNativeDriver: boolean,
+  animationConfig: {|
+    friction?: number,
+    tension?: number,
+    speed?: number,
+    bounciness?: number,
+  |},
+
+  dispatch: Dispatch,
+  ...SelectorProps,
+|}>;
+
+class ProgressBar extends PureComponent<Props> {
```

Comment:
Could you rename this file and class? It's not really a "progress" bar (since we're not measuring progress), I think "LoadingBar" would be better.

Reply:
ok , It will be better, will do it